NDTensors/test/ITensors/TestITensorDMRG/TestITensorDMRG.jl

```diff
@@ -17,9 +17,9 @@ is_broken(dev, elt::Type, conserve_qns::Val) = false
 
 ## Disable blocksparse GPU testing on CUDA, Metal and ROC backends.
 ## Currently Metal fails because we are waiting for `resize!` to be added. Should be in the next metal release
-## CUDA fails because there is no defined `append!`.
 ## ROC fails because TODO determine again why roc fails.
-is_broken(dev::typeof(cu), elt::Type, conserve_qns::Val{true}) = true
+
+#is_broken(dev::typeof(cu), elt::Type, conserve_qns::Val{true}) = true
```

Comment:
Reminder to delete this before merging.

Reply:
Metal fails because of the `resize!` problem but I removed the CUDA and ROCM functions

Reply:
@mtfishman should I make an `append!` for Metal which converts it to CPU and then back to the GPU?

Reply:
I'm not sure if that is possible to do in-place right now with having something like `resize!`.

Reply:
Good point I forgot about it having to be in-place. I added a comment in-line. Thanks!